golos.publication/golos.publication.abi

```diff
@@ -614,6 +624,96 @@
             "type": "limitsarg"
         }
     ]
+  },
+  {
+    "name": "st_max_vote_changes",
+    "base": "",
+    "fields": [
+      {
+        "name": "max_vote_changes",
+        "type": "uint8"
+      }
+    ]
+  },
+  {
+    "name": "st_cashout_window",
+    "base": "",
+    "fields": [
+      {
+        "name": "cashout_window",
+        "type": "uint32"
+      }
+    ]
+  },
+  {
+    "name": "st_upvote_lockout",
+    "base": "",
+    "fields": [
+      {
+        "name": "upvote_lockout",
+        "type": "uint8"
```

Comment:
`uint8` is not enough, it can hold up to 255 seconds, which is too low. Make this `uint32`